docs/hub-deployment-guide/configure-auth/cilogon.md

```diff
@@ -149,7 +149,7 @@ The steps to enable the JupyterHub CILogonOAuthenticator for a hub are simmilar
               CILogonOAuthenticator:
                 oauth_callback_url: https://{{ HUB_DOMAIN }}/hub/oauth_callback
                 shown_idps:
-                  - https://github.com/login/oauth/authorize
+                  - http://github.com/login/oauth/authorize
```

Comment:
This change is to have the urls for `shown_idps` and `allowed_idps` be the same to avoid confusion, right?

Reply:
Yepp! I've verified this to work as well!